docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+services:
+  web:
+    build: ./src/
+    ports:
+      - "8080:80"
```

Comment:
This makes the web container available on port `8080` because you _probably_ want a firewall in front of this.